NOTICES.txt

```diff
@@ -3978,7 +3978,7 @@ Legal Terms
 
 ------------------------------------
 
-Package path: ..\..\..\3rdParty\packages\googlebenchmark-1.5.0-rev2-windows\googlebenchmark\LICENSE

```

Comment:
I never updated `NOTICES.txt`, `SPDX-Licenses.csv` or `SPDX-Licenses.json` files myself. I think they are autogenerated or there is a process to autogenerate them? Tagging @lawsonamzn to confirm.

Reply:
@lawsonamzn are these files required to be updated each time or is it part of an automatic process? Thanks.

Reply:
@lawsonamzn @spham-amzn Do you know the answer to this? Is it required to manually update it each time or is it an automatic process?

Reply:
It's not an auto-generated and needs to be updated by hand. Looks like SPDX-Licenses.* needs to be updated as well.  @amzn-changml may know

Reply:
Confirmed with @amzn-changml it's a manual process at the moment. Eventually it's going to be a github action that triggers once something is merged into the 3p-package-source.

Reply:
Good to know! Thanks!